controlplane/kubeadm/api/v1alpha3/conversion.go

```diff
@@ -68,9 +68,11 @@ func (dest *KubeadmControlPlaneList) ConvertFrom(srcRaw conversion.Hub) error {
 }
 
 func Convert_v1alpha4_KubeadmControlPlaneSpec_To_v1alpha3_KubeadmControlPlaneSpec(in *v1alpha4.KubeadmControlPlaneSpec, out *KubeadmControlPlaneSpec, s apiconversion.Scope) error {
+	in.RolloutAfter = out.UpgradeAfter
```

Comment:
I'm probably missing something, but don't we have to do this the other way around?
i.e.
```go
out.UpgradeAfter = in.RolloutAfter
```
(same below)

Reply:
Hm yes, I wonder why the tests didn't catch this though, we have fuzzy tests that should 🤔